AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/EventManagerInviteController.cs

```diff
@@ -68,16 +70,33 @@ public EventManagerInviteController(IMediator mediator, UserManager<ApplicationU
                 }
 
                 var userToInvite = await _userManager.FindByEmailAsync(invite.InviteeEmailAddress);
-                if (userToInvite != null && userToInvite.ManagedEvents.Any(m => m.EventId == eventId))
+                if (userToInvite?.ManagedEvents != null && userToInvite.ManagedEvents.Any(m => m.EventId == eventId))
                 {
                     ModelState.AddModelError("InviteeEmailAddress", $"{invite.InviteeEmailAddress} is allready a manager for this event");
                     return View("Send", invite);
                 }
 
                 invite.EventId = eventId;
                 var user = await _userManager.GetUserAsync(User);
-                await _mediator.SendAsync(new CreateEventManagerInviteCommand { Invite = invite, UserId = user.Id });
+                var inviteId = await _mediator.SendAsync(new CreateEventManagerInviteCommand { Invite = invite, UserId = user.Id });
 
+                await _mediator.PublishAsync(new SendEventManagerInvite
```

Comment:
Moving events inside of the trigger that causes them prevents having to publish them from multiple locations.